### PR TITLE
added peer dependencies to one click lca widget

### PR DESCRIPTION
**Pull request by RytisSapka**

### CODE REVIEW

File: packages/itwin/one-click-lca-widget/package.json [hunk 1]
```diff
@@ -43,14 +43,25 @@
...
   "devDependencies": {
-    "@itwin/appui-abstract": "^3.0.0",
-    "@itwin/appui-react": "^3.0.0",
-    "@itwin/build-tools": "^3.0.0",
-    "@itwin/certa": "^3.0.0",
-    "@itwin/core-common": "^3.0.0",
-    "@itwin/core-frontend": "^3.0.0",
-    "@itwin/core-react": "^3.0.0",
-    "@itwin/eslint-plugin": "^3.0.0",
+    "@itwin/appui-abstract": "^3.1.2",
```
Comment (Guillar1) [inline packages/itwin/one-click-lca-widget/package.json:46]: I believe ^3.0.0 is good enough for all of these.

File: packages/itwin/one-click-lca-widget/package.json [hunk 2]
```diff
@@ -73,17 +84,32 @@
...
     "rimraf": "^3.0.2",
     "ts-jest": "^27.1.3",
     "typescript": "~4.3.0"
   },
   "peerDependencies": {
-    "@itwin/appui-abstract": "^3.0.0",
-    "@itwin/appui-react": "^3.0.0",
-    "@itwin/core-frontend": "^3.0.0",
-    "@itwin/core-react": "^3.0.0",
+    "@itwin/appui-abstract": "^3.1.2",
```
Comment (aruniverse) [inline packages/itwin/one-click-lca-widget/package.json:94]: are all of these actually peer deps? or did you just add whatever dev deps to here?
transitive peers that are missing, which are reported by the `rush update` should be added as dev deps.
you guys should review these peer deps, if theyre only being used to import types then they can remain as dev deps. and any pure/effect-free usage of classes can also probably be a dev dep

File: packages/itwin/one-click-lca-widget/package.json (hunk 1 above)
Comment (RytisSapka) [reply packages/itwin/one-click-lca-widget/package.json]: Bumped it back down to 3.0.0
Not sure why, but at first rush was giving warnings about unmet peer dependencies for all these @itwin packages and the warnings went away after bumping the versions up to 3.1.2.
Now that I reverted to 3.0.0 the warnings did not reappear and everything seems to be in order.

File: packages/itwin/one-click-lca-widget/package.json (hunk 2 above)
Comment (RytisSapka) [reply packages/itwin/one-click-lca-widget/package.json]: Yeah, my bad. I added all the dependencies reported by ```rush update``` to peer dependencies and dev dependencies. Guillermo took a look and decided that the original peer dependencies were good so I removed what I added.

**Comment by aruniverse** [comment]
missing rush change

File: common/changes/@itwin/reports-config-widget-react/oclca-peer-dependencies_2022-09-21-20-46.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
```
Comment (aruniverse) [inline common/changes/@itwin/reports-config-widget-react/oclca-peer-dependencies_2022-09-21-20-46.json:3]: dont need this file

File: common/changes/@itwin/tree-widget-react/oclca-peer-dependencies_2022-09-21-20-46.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/tree-widget-react",
```
Comment (aruniverse) [inline common/changes/@itwin/tree-widget-react/oclca-peer-dependencies_2022-09-21-20-46.json:4]: dont need this file

**Review by aruniverse — APPROVED** [review]

File: common/changes/@itwin/reports-config-widget-react/oclca-peer-dependencies_2022-09-21-20-46.json (hunk 3 above)
Comment (RytisSapka) [reply common/changes/@itwin/reports-config-widget-react/oclca-peer-dependencies_2022-09-21-20-46.json]: deleted

File: common/changes/@itwin/tree-widget-react/oclca-peer-dependencies_2022-09-21-20-46.json (hunk 4 above)
Comment (RytisSapka) [reply common/changes/@itwin/tree-widget-react/oclca-peer-dependencies_2022-09-21-20-46.json]: deleted

**Pull request merged**